pull_request.rb

```diff
@@ -0,0 +1,8 @@
+# Adding this text for testing pull_requests and comments
+
+
+# one more line for testing
+
+# one more line
```

Comment:
this line will be deleted in next commit

Reply:
test comment